test/middleware.test.js

```diff
@@ -0,0 +1,99 @@
+jest.mock('isomorphic-fetch')
+
+import { successUrl, failureUrl } from 'isomorphic-fetch'
+import { middleware, LP_API } from '../src'
+import { lpApiRequest, lpApiSuccess, lpApiFailure } from '../src/actions'
+
+const configuredMiddleware = middleware({})
+
+/* HELPERS */
+
+// Pass action to middleware, and wait for the specified number of actions 
+// to be passed to the next middleware before resolving
+const callMiddleware = (action, opt={}) => {
```

Comment:
💥  just blew my mind

You can also checkout http://redux.js.org/docs/recipes/WritingTests.html#middleware for other ways to test middleware